### PR TITLE
✨ Adapt for bison fute

### PR DESCRIPTION
**Pull request by BertrandD**
- **🎨 Run prettier**
- **✨ Adapt for bison fute**

### CODE REVIEW

File: src/explorer/TransactionView.vue [hunk 1]
```diff
@@ -1,71 +1,77 @@
...
-
-const txData = computed(() => transactionsStore.value.transactionData[txHash.value]);
-const parsedTx = computed(() => getParsedTx(txData.value));
-
-const targetTx = computed(() => {
-    if (!txData.value) return '';
-    return Array.from(getParsedTx<MsgPublishPayloadProof>(txData.value).txHash).map(x => x.toString(16).padStart(2, '0')).join('');
-})
+const txData = computed(() => {
+    console.log("aze", txHash.value);
```
Comment (maxgttph) [inline src/explorer/TransactionView.vue:15]: aze ?

File: src/explorer/ContractView.vue [hunk 2]
```diff
@@ -16,12 +16,18 @@ const transactions = computed(() =>
...
+});
 
 function parseBase64(base64: string): string {
-    if (!base64) return "";
-    const bytes = atob(base64);
-    return [...bytes].map((byte) => byte.charCodeAt(0).toString(16).padStart(2, "0")).join("");
+    return base64;
+    //if (!base64) return "";
+    //const bytes = atob(base64);
+    //return [...bytes].map((byte) => byte.charCodeAt(0).toString(16).padStart(2, "0")).join("");
```
Comment (maxgttph) [inline src/explorer/ContractView.vue:30]: still intended to be commented

**Review by maxgttph — APPROVED** [review]

File: src/explorer/ContractView.vue [hunk 3]
```diff
@@ -16,12 +16,18 @@ const transactions = computed(() =>
     Object.values(transactionsStore.value.transactionData).filter((tx) => tx.contracts?.includes(contract_name.value)),
 );
 
-const contractData = computed(() => contractsStore.value.contractData?.[contract_name.value]);
+const contractData = computed(() => {
+    console.log("aze", contract_name.value);
+    console.log("store", JSON.stringify(contractsStore.value.contractData));
+    console.log("data", contractsStore.value.contractData[contract_name.value]);
```
Comment (wraitii) [inline src/explorer/ContractView.vue:22]: autre `aze`

**Review by wraitii — APPROVED** [review]
Enlever les `console.log` et fuego

**Comment by wraitii** [comment]
Closing, will redo

**Pull request closed without merging**